src/spectrecoff-cli/commands/CanvasImage.fs

```diff
@@ -33,38 +33,35 @@ type CanvasImageExample() =
         |> Async.RunSynchronously
         0
 
+open SpectreCoff.Cli.Documentation
+
 type CanvasImageDocumentation() =
     inherit Command<CanvasImageSettings>()
     interface ICommandLimiter<CanvasImageSettings>
 
     override _.Execute(_context, _settings) =
-        Cli.Theme.setDocumentationStyle
-        BL |> toConsole
-        pumped "CanvasImage module"
-        |> alignedRule Left
-        |> toConsole
+        setDocumentationStyle
+        docHeader "CanvasImage module" |> toConsole
```

Comment:
I think you might have overlooked this:

![image](https://github.com/EluciusFTW/SpectreCoff/assets/17160319/6e3199cd-016f-44ca-9122-18428ed51bb8)

Reply:
Good catch, removed.